server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/ICSAttachmentWorkflowTest.java

```diff
@@ -491,6 +492,10 @@ public void setup() throws Exception {
                             .addProperty("destination", JSON_MAIL_ATTRIBUTE)
                             .build())
                     .addMailet(MailetConfiguration.builder()
+                        .match("All")
+                        .clazz("org.apache.james.jmap.mailet.ICALBodyTextToAttachment")
```

Comment:
Why do we use FQDN? ICALBodyTextToAttachment should be enough

Reply:
Tried but it did not work because this mailet belongs to Jmap Mailet. The same as "VacationMailet"